src/index.ts

```diff
@@ -730,6 +731,16 @@ class Datastore extends DatastoreRequest {
     return Datastore.isKey(value);
   }
 
+  keyToLegacyUrlsafe(key: entity.Key, locationPrefix?: string): Promise<string>;
```

Comment:
In these other places, I believe we need to now update to the new camelcase system, `UrlSafe`/`urlSafe`

Reply:
Rename the method name, also updated to description.